File: A's/cpp/158A.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int n, k; cin >> n >> k;

    vector<int> v;

    while (n--) {
        int w; cin >> w;
        v.push_back(w);
    }

    int score = 0;
    int pass_score = v.at(k - 1);
    
    for (int i = 0; i < v.size(); i++) {
        if (v.at(i) >= pass_score && v.at(i) > 0) {
            score++;
        }
    }
    cout << score << endl;
    
    return 0;
}

File: A's/py/158A.py
def main() -> None:
    n, k = map(int, input().split())

    v: list[int] = list(map(int, input().split()))

    score: int = 0
    pass_score: int = v[k - 1]

    for s in v:
        if s >= pass_score and s > 0:
            score += 1

    print(score)

if __name__ == "__main__":
    main()